rpcs3/Emu/RSX/GL/GLTexture.cpp

```diff
@@ -442,8 +443,43 @@ namespace gl
 		}
 	}
 
+	void apply_swizzle_remap(const GLenum target, const std::array<GLenum, 4>& swizzle_remap, const std::pair<std::array<u8, 4>, std::array<u8, 4>>& decoded_remap)
+	{
+		//Remapping tables; format is A-R-G-B
+		//Remap input table. Contains channel index to read color from
+		const auto remap_inputs = decoded_remap.first;
+
+		//Remap control table. Controls whether the remap value is used, or force either 0 or 1
+		const auto remap_lookup = decoded_remap.second;
+
+		GLenum remap_values[4];
+
+		for (u8 channel = 0; channel < 4; ++channel)
```

Comment:
using `u8` just because the loop variable fits in range isn't really beneficial and can have negative performance (especially if it's used in indexing). plain `int` /`unsigned` is usually faster. Also, you can use something like `int_fast8_t`.